spec/features/issues_spec.rb

```diff
@@ -323,112 +323,6 @@
           include_examples 'deleted item is listed in Trash', :issue
           include_examples 'recover deleted item', :issue
         end
-
-        describe 'add evidence', js: true do
```

Comment:
you're removing significant amounts of test coverage without replacing it with other coverage.

Are these specs already elsewhere?

Reply:
The specs are bring migrated/updated in the add evidence PR. New specs are also being added in the bulk delete PR. Both PR's will be merged into this one so we will not be without test coverage once this is ready to merge to develop